gustaf/io/nutils.py

```diff
@@ -0,0 +1,147 @@
+"""gustaf/gustaf/io/nutils.py
+
+io functions for nutils.
+"""
+import numpy as np
+
+from gustaf.faces import Faces
+from gustaf.volumes import Volumes
+from gustaf.io.ioutils import abs_fname, check_and_makedirs
+from gustaf.utils import log
+from gustaf.io import mixd
+
+
+def load(fname):
+    """nutils load.
+    Loads a nutils (np.savez) file and returns a Gustaf Mesh.
+
+    Parameters
+    -----------
+    fname: str
+      The npz file needs the following keys:
+      nodes, cnodes, coords, tags, btags, ptags.
+    """
+    npzfile = np.load(fname, allow_pickle=True)
+    nodes = npzfile['nodes']
+    _ = npzfile['cnodes']
+    coords = npzfile['coords']
+    _ = npzfile['tags'].item()
+    btags = npzfile['btags'].item()
+    _ = npzfile['ptags'].item()
+
+    vertices = coords
+
+    # connec
+    simplex = True
+    connec = None
+
+    if vertices.shape[1] == 2:
+        volume = False
+    else:
+        volume = True
+
+    try:
+        connec = nodes
+    except BaseException:
+        log.debug("Error")
```

Comment:
Could you expand on the error? Or add a comment on why this error can be ignored in this case?

Reply:
This Error was a bit unnecessary here, I'm going to change it to make it more clear.